packages/itwin/grouping-mapping-widget/src/widget/components/customUI/ManualGroupingCustomUI.tsx

```diff
@@ -6,9 +6,9 @@ import React from "react";
 import { Button, LabeledTextarea, Text } from "@itwin/itwinui-react";
 import { LoadingSpinner } from "../utils";
 import "./ManualGroupingCustomUI.scss";
-import type { GroupingCustomUIProps } from "./GroupingMappingCustomUI";
+import type { GroupingCustomUIProps } from "./groupingMappingCustomUI";
```

Comment:
why was this file name changed to start with lower case? ❓

Reply:
Some file names were inconsistent. I undid it. I'll leave it for another PR.